packages/flutter/lib/src/material/text_theme.dart

```diff
@@ -82,9 +82,9 @@ class TextTheme with Diagnosticable {
   /// If you do decide to create your own text theme, consider using one of
   /// those predefined themes as a starting point for [copyWith] or [apply].
   ///
-  /// Please note that you can not mix and match the 2018 styles with the 2021
-  /// styles. Only one or the other is allowed in this constructor. The 2018
-  /// styles will be deprecated and removed eventually.
+  /// The 2018 styles cannot be mixed with the 2021 styles. Only one or the
+  /// other is allowed in this constructor. The 2018 styles are deprecated and
+  /// will eventually be removed.
```

Comment:
The American school system is failing me here, but adverb should be after the verb right?

```suggestion
  /// will be removed eventually.
```

Reply:
either is fine as far as i'm aware

Reply:
Ah gotcha, feel free to ignore then :)